docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -189,6 +189,7 @@ apply to already started executions. Defaults to false.
 
 ### Jobs History Limits
 
-The `.spec.successfulJobsHistoryLimit` and `.spec.failedJobsHistoryLimit` fields are optional. These fields specify how many completed and failed jobs should be kept.
```

Comment:
Is the statement in "Deleting a CronJob" section (L107) still true? 

```
This stops new jobs from being created. However, running jobs won't be stopped, 
and no jobs or their pods will be deleted.
```

IIUC it's now fixed and `kubectl delete cronjob` now deletes its jobs and pods.

This can be a separate PR (and target `master` branch).